src/formatDate.js

```diff
@@ -50,7 +50,24 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
+  const toSeparator = toFormat.pop();
```

Comment:
`pop()` is a mutating method and we should not mutate arguments of function

Reply:
Not fixed